admin_guide/quota.adoc

```diff
@@ -320,7 +351,7 @@ spec:
 <1> The total number of pods in a non-terminal state with *BestEffort* quality
 of service that can exist in the project.
 <2> Restricts the quota to only matching pods that have *BestEffort* quality of
-service for either memory or CPU.
+service for either memory, CPU, or ephemeral storage.
```

Comment:
`ephemeral-storage` is not tied to QoS.

https://github.com/kubernetes/kubernetes/blob/master/pkg/apis/core/helper/qos/qos.go#L37

Reply:
Yes, I was cleaning things up and missed that one.